packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/shippingRadios.css

```diff
@@ -10,7 +10,7 @@
 
 .radioLabel {
     display: grid;
-    grid-template-columns: min-content auto;
+    grid-template-columns: minmax(auto, max-content) auto;
```

Comment:
I believe `minmax(auto, max-content)` is identical to `minmax(auto, auto)`, which is identical to `auto`. See [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/minmax).

> **`auto`**
>
> As a maximum, identical to max-content.

Reply:
Resolved in https://github.com/magento/pwa-studio/pull/2280/commits/40211e9bf02e4242fea36e8136aa78b4d2ea8d90.